static/app/views/alerts/rules/issue/ticketRuleModal.spec.tsx

```diff
@@ -149,28 +150,72 @@ describe('ProjectAlerts -> TicketRuleModal', function () {
       await submitSuccess();
     });
 
-    it('should ignore error checking when default is empty array', function () {
-      renderComponent({
-        otherField: {
-          label: 'Labels',
-          required: false,
-          choices: [['bug', `bug`]],
-          default: [],
-          type: 'select',
-          multiple: true,
-          name: 'labels',
-        },
+    it('should ignore error checking when default is empty array', async function () {
```

Comment:
This test was passing, although I was passing the arguments in badly and did not actually test anything before. Fixed it by correcting arguments to `renderComponent` and actually submitting modal